### DISCUSSION TITLE
Virus hit?

### DISCUSSION THREAD

**Discussion by Firebat-15** [lacymorrow/crossover#381] [Q&A] [answered]
Is there a reason this flags in virus scanners? is it just the way its designed or should i be worried?

**Comment by lacymorrow** [comment, marked as answer]
Electron (the framework I used to build CrossOver) doesn't allow the app to listen to key/mouse presses (especially single-key presses) when it isn't the active app. The method that I use to allow the app to respond to mouse key presses and keyboard combos gets flagged as a virus because of the native C library it uses, `libiohook`.

CrossOver uses a node wrapper [wilix-team/iohook](https://github.com/wilix-team/iohook) to handle input listeners, all of the code involved is located here in this file:
https://github.com/lacymorrow/crossover/blob/master/src/main/iohook.js

You may verify the code you run on your own PC by following the Development instructions in the README and building for windows with `yarn build:windows:exe`

Thank you for bringing this up.
